content/syllabuses/web-dev.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Web Development
+---
+
+- Introduction to NCIT curriculum
+- NCIT 01: Conducting research and user interviews
+- NCIT 02: Reporting
+- NCIT 03: Research presentation
+- {{% contentlink "topics/agile-scrum" %}}
+- {{% contentlink "workshops/agile-scrum/" %}}
+- NCIT 18: Agile project management
+- NCIT 19: Agile meetings
+- {{% contentlink "workshops/how-to-be-a-professional" %}}
+- {{% contentlink "projects/linux/beginner" %}}
+- {{% contentlink "topics/web-dev-self-learning" %}}
+- {{% contentlink "topics/clean-code" %}}
+- {{% contentlink "projects/git-exercises" %}}
+- NCIT 04: Version control (GIT) for teamwork
+- {{% contentlink "projects/first-personal-website" %}}
+- {{% contentlink "topics/how-to-ask-for-help-with-your-code" %}}
+- {{% contentlink "workshops/big-picture-web-dev" %}}
+- NCIT 10: Workshop: Giving and receiving constructive feedback
+- NCIT 06: Web design for business
+- NCIT 07: Building an online business
+- NCIT 12: Business and technology
+- NCIT 09: Ethics and Professionalism in IT
+- {{% contentlink "projects/basic-flow-control-katas" %}} 
+- {{% contentlink "topics/js-and-node-specific/super-basic-intro-to-node" %}}
+- WORKSHOP: Live code. Super basic intro to node
+{{% contentlink "topics/unit-testing" %}}
+%}}: add some tests to existing code
+- {{% contentlink "topics/test-driven-development" %}}
+- {{% contentlink "workshops/intro-to-unit-testing-with-jasmine" %}}
+- {{% contentlink "workshops/js-and-node-specific/basic-intro-to-oop" %}}
+- {{% contentlink "topics/js-and-node-specific/oop-basics" %}}
+- {{% contentlink "projects/oop/animals/part2" 
+- NCIT 13: Object-oriented programming
+- {{% contentlink "topics/how-the-internet-works" %}}
+- {{% contentlink "topics/data-ethics-and-privacy" %}}
+- {{% contentlink "projects/basic-flow-control-katas-assertive" %}}
```

Comment:
- {{% contentlink "projects/basic-flow-control-katas-assertive" %}}

This is in a funny place, if anything it should be before unit tests. It might actually be a precursor even